### DISCUSSION TITLE
mqtt, after Home Assistant update, mqtt sensors are no longer working

### DISCUSSION THREAD

**Discussion by HA-TB303** [dingo35/SmartEVSE-3.5#61] [General]
Hi, we are having the issue that our 4 chargers are no longer sending data over mqtt after a home assistant update, or a HA container restart.

The chargers are in a different subnet, but the mqtt server is available without issues for these 4 chargers, so that does not seem to be the issue. The only way to resolve the issue, is logging in to the smartevse web-UI and manually reboot them. After that mqtt is working agian.

**Comment by dingo35** [comment]
Are you running v3.5.1?

**Comment by HA-TB303** [comment]
No I'm still on v3.5.0-RC1. Is there a related change or fix in that version?

I'll update 2 chargers anyway...

**Comment by dingo35** [comment]
I can't reproduce it here, BUT I have HA running "native", not in a container. Containerized installs if HA behave somewhat differently because of the more complicated networking.

**Comment by dingo35** [comment]
You can also try this alpha 3.6 version, the MQTT library of this one is completely replaced by mongoose MQTT....
[b938b7af.zip](https://github.com/dingo35/SmartEVSE-3.5/files/15092371/b938b7af.zip)

...no spiffs.bin needed in this version....

**Comment by HA-TB303** [comment]
I updated 2, restarted the Home Assistent VM and those 2 came online, the other 2 did not.

I have now updated all 4 tot 3.5.1 and all 4 came online after the VM restart. I think you did fix the issue even without knowing so ;)

I'll try 3.6 as well on one of the chargers.

**Comment by HA-TB303** [comment]
With version 3.6a, no issues either :+1:

**Comment by dingo35** [comment]
Ok great!
